packages/snippet-bot/test/snippet-bot.ts

```diff
@@ -14,6 +14,7 @@
 //
 
 /* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
We really try to avoid that one :)

Reply:
I changed it to `eslint-disable-next-line`. Currently I don't know how to fix it.